src/playground/components/ShareURL.js

```diff
@@ -89,6 +89,34 @@ export default function ShareURL({ url }) {
 							</span>
 						)}
 					</div>
+
+					<button
+						onClick={() => {
+							const reportUrl = new URL(
+								"https://github.com/eslint/eslint/issues/new",
+							);
+							const params = {
+								labels: "bug,repro:yes",
+								template: "bug-report.yml",
+								"repro-url": window.location,
```

Comment:
```suggestion
								"repro-url": window.location.href,
```
We only need the URL, not the entire `window.location` object I guess.